ui/pages/confirm-import-token/confirm-import-token.test.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import reactRouterDom from 'react-router-dom';
+import configureMockStore from 'redux-mock-store';
```

Comment:
Thank you for bringing this up! From the same link you posted, the way to test Redux is to

> render a <Provider> with a real store instance wrapping the component/s being tested.

I am with you. I agree that using a mock store is not necessary. I think we should move forward using the real Redux store to preserve the integrity of our integration tests. It would be nice to deprecate the 49 `redux-mock-store` instances in this repo as we come across them and move onto RTL and hook-based components.

Made the change